.travis.helm-publish.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+riff_version=`cat VERSION`
+helm_charts_bucket='riff-charts'
+helm_charts_url="https://${helm_charts_bucket}.storage.googleapis.com/"
+work_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helm-charts/.work"
+
+if [ "$GCLOUD_CLIENT_SECRET" != ""]; then
```

Comment:
should this be:

`if [[ "$GCLOUD_CLIENT_SECRET" != "" ]]; then`

my bash complained

Reply:
done